### PR TITLE
Add Nick to translations

### PR DESCRIPTION
**Pull request by vincentiotex**

### CODE REVIEW

File: en.yaml [hunk 1]
```diff
@@ -127,6 +127,11 @@ home.monika.desc.keep: >-
   privacy, networking and emerging technologies.
 home.monika.name.keep: Monika Hathaway
 home.monika.title.keep: Communication
+home.nick.desc.keep: >-
+  Full stack intern. Master in Computer Science at USC and Bachelor in CS at
+  University of Wisconsin-Madison.  Machinel learning enthusiast.
```
Comment (raullenchai) [inline en.yaml]: --s/Machinel/Machine
--also blockchain enthusiast?
-- Bachelor at ...

File: ko.yaml [hunk 2]
```diff
@@ -108,6 +108,11 @@ home.larry.title.keep: Growth & Community
 home.monika.desc.keep: '사이버 보안, 개인정보 보호, 네트워킹 및 신흥 기술 분야에서 깊이 있는 전문지식을 갖춘 기술 통신 전문가.'
 home.monika.name.keep: Monika Hathaway
 home.monika.title.keep: 통신
+home.nick.desc.keep: >-
+  Full stack intern. Master in Computer Science at USC and Bachelor in CS at
```
Comment (raullenchai) [inline ko.yaml]: should in Korean?

File: zh-CN.yaml [hunk 3]
```diff
@@ -83,6 +83,11 @@ home.larry.title.keep: 增长与社区
 home.monika.desc.keep: 专注于技术类媒体传播，在网络安全、隐私、网络和新兴技术方面拥有丰富专业知识。
 home.monika.name.keep: Monika Hathaway
 home.monika.title.keep: 公众传播
+home.nick.desc.keep: >-
+  Full stack intern. Master in Computer Science at USC and Bachelor in CS at
```
Comment (raullenchai) [inline zh-CN.yaml]: Chinese?

File: ko.yaml (hunk 2 above)
Comment (vincentiotex) [reply ko.yaml]: @raullenchai , are we supposed to translate them ourselves? Or do we wait for the community to do it ?

File: en.yaml (hunk 1 above)
Comment (vincentiotex) [reply en.yaml]: @niickchen , you can fix this

File: zh-CN.yaml (hunk 3 above)
Comment (niickchen) [reply zh-CN.yaml]: Added

**Pull request merged**